### DISCUSSION TITLE
Upgrading past beta.4 breaks Jest

### DISCUSSION THREAD

**Discussion by davetapley** [convert-units/convert-units#343] [Q&A]
I _assume_ I'm doing something weird, but posting here because I have no idea in case others have it / can help:

I'm using [craco](https://github.com/dilanx/craco) and since upgrading from `beta.4` to `beta.6` (and updating imports for #256) I'm getting:
```
Cannot find module 'convert-units/definitions/length'
```

But **only** when I `npm run test`. It's happening on CI and on a new build.
Both `npm start` and `npm build` work fine.

Any ideas welcome.

**Comment by Taar** [comment]
Without additional context I'm not sure I can be of much help here. However, my gut instinct is that the tests are using a different method for finding packages. Perhaps [my comment here](https://github.com/convert-units/convert-units/issues/327#issuecomment-2329569828) might be helpful?

**Reply by davetapley** [reply]
Thanks, since `"type": "module"` I'm running in to this, so I'll diagnose that and report back:
- https://github.com/dilanx/craco/issues/408
